### PR TITLE
Added myself to members page

### PR DESCRIPTION
**Pull request by csteamengine**
### Problem / Feature
I really effed up the first time. I started over.
### Solution
I started over and added myself and didn't touch a single other file.
### What To Test
Make sure I'm not an idiot.
### Reviewers
EVERYONE!!!! :)
@REVIEWER

FYI: @KrashLeviathan @csteamengine @regalcat

### CODE REVIEW

File: members/members.json [hunk 1]
```diff
@@ -12,7 +16,7 @@
       "image": "../_images/members/judge-jeffrey-md.jpeg",
       "description": "Freshman in Computer Science"
     },
-    "Michielu Menning": {
+    "MichieluMenning": {
```
Comment (KrashLeviathan) [inline members/members.json]: Lol why did you remove the space between his first and last name? Other than that, everything looks good.

Comment (csteamengine) [inline members/members.json]: Not gonna lie, I don't recall that.

**Comment by KrashLeviathan** [comment]
+1
